UI/MapWnd.cpp

```diff
@@ -2753,13 +2753,37 @@ void MapWnd::InitTurn() {
     // connect system fleet add and remove signals
     for (auto& system : objects.all<System>()) {
         m_system_fleet_insert_remove_signals[system->ID()].push_back(system->FleetsInsertedSignal.connect(
-            boost::bind(&MapWnd::FleetsInsertedSignalHandler, this, _1)));
+            [this](const std::vector<std::shared_ptr<Fleet>>& fleets) {
```

Comment:
is this relevant to this commit?

Reply:
I don't object, but it seems out of place given the commit message...

Reply:
Yes, changing to boost::range would have required to update the method signature of the now-removed methods. As those were only called here I moved the logic into this place to avoid figuring out the correct signature for those methods.